File: examples/js_vs_python/controlflow.py
# https://en.wikipedia.org/wiki/Fizz_buzz
for i in range(20):
    div_by_3 = i % 3 == 0
    div_by_5 = i % 5 == 0
    if div_by_3 and div_by_5:
        print("Fizz Buzz")
    elif div_by_3:
        print("Fizz")
    elif div_by_5:
        print("Buzz")
    else:
        print(i)

File: examples/js_vs_python/controlflow.js
// https://en.wikipedia.org/wiki/Fizz_buzz
for (let i = 0; i < 20; i += 1) {
  const divBy3 = i % 3 === 0;
  const divBy5 = i % 5 === 0;
  if (divBy3 && divBy5) {
    console.log("Fizz Buzz");
  } else if (divBy3) {
    console.log("Fizz");
  } else if (divBy5) {
    console.log("Buzz");
  } else {
    console.log(i);
  }
}
